app/transport/rest/user_test.go

```diff
@@ -0,0 +1,112 @@
+package rest
+
+import (
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/go-devs-ua/octagon/app/entities"
+	"github.com/go-devs-ua/octagon/app/globals"
+	"github.com/go-devs-ua/octagon/lgr"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUserHandler_DeleteUser(t *testing.T) {
+	logger, err := lgr.New("INFO")
```

Comment:
Maybe it's better to replace "INFO" with a constant from the lgr package?

Reply:
Ok, done